mono/metadata/profiler.h

```diff
@@ -30,7 +30,8 @@ typedef enum {
 	MONO_PROFILE_MONITOR_EVENTS   = 1 << 17,
 	MONO_PROFILE_IOMAP_EVENTS     = 1 << 18, /* this should likely be removed, too */
 	MONO_PROFILE_GC_MOVES         = 1 << 19,
-	MONO_PROFILE_GC_ROOTS         = 1 << 20
+	MONO_PROFILE_GC_ROOTS         = 1 << 20,
+	MONO_PROFILE_CONTEXT_EVENTS   = 1 << 21
```

Comment:
Add a comma here, it will avoid line damage the next time we add an entry to this enum.